--- Isabelle2014-orig/lib/Tools/java	2014-08-27 18:34:13.000000000 +0900
+++ Isabelle2014/lib/Tools/java	2015-01-10 16:55:00.186627226 +0900
@@ -10,5 +10,6 @@ declare -a JAVA_ARGS; eval "JAVA_ARGS=($
 unset CLASSPATH
 
 isabelle_jdk java "${JAVA_ARGS[@]}" \
-  -classpath "$(jvmpath "$ISABELLE_CLASSPATH")" "$@"
+  -classpath "$(jvmpath "$ISABELLE_CLASSPATH")" \
+  "-Djava.ext.dirs=$(jvmpath "$ISABELLE_JAVA_EXT:$ISABELLE_HOME/lib/classes/ext:$(java-config -p scala | sed 's@\([^:]*\)/[^:]*:.*@\1@')")" "$@"
 
diff -rupN Isabelle2014-orig/lib/Tools/scala Isabelle2014/lib/Tools/scala
--- Isabelle2014-orig/lib/Tools/scala	2014-08-27 18:34:13.000000000 +0900
+++ Isabelle2014/lib/Tools/scala	2015-01-10 16:55:00.186627226 +0900
@@ -7,5 +7,6 @@
 isabelle_admin_build jars || exit $?
 
 isabelle_scala scala -Dfile.encoding=UTF-8 \
-  -classpath "$(jvmpath "$ISABELLE_CLASSPATH")" "$@"
+  -classpath "$(jvmpath "$ISABELLE_CLASSPATH")" \
+  "-Djava.ext.dirs=$(jvmpath "$ISABELLE_JAVA_EXT:$ISABELLE_HOME/lib/classes/ext:$(java-config -p scala | sed 's@\([^:]*\)/[^:]*:.*@\1@')")" "$@"
 
diff -rupN Isabelle2014-orig/lib/Tools/scalac Isabelle2014/lib/Tools/scalac
--- Isabelle2014-orig/lib/Tools/scalac	2014-08-27 18:34:13.000000000 +0900
+++ Isabelle2014/lib/Tools/scalac	2015-01-10 16:55:00.186627226 +0900
@@ -7,5 +7,6 @@
 isabelle_admin_build jars || exit $?
 
 isabelle_scala scalac -Dfile.encoding=UTF-8 \
-  -classpath "$(jvmpath "$ISABELLE_CLASSPATH")" "$@"
+  -classpath "$(jvmpath "$ISABELLE_CLASSPATH")" \
+  "-Djava.ext.dirs=$(jvmpath "$ISABELLE_JAVA_EXT:$ISABELLE_HOME/lib/classes/ext:$(java-config -p scala | sed 's@\([^:]*\)/[^:]*:.*@\1@')")" "$@"

--- Isabelle2014-orig/contrib/jfreechart-1.0.14/etc/settings	2012-12-09 07:23:53.000000000 +1100
+++ Isabelle2014/contrib/jfreechart-1.0.14/etc/settings	2014-08-18 16:18:10.244698000 +1000
@@ -1,7 +1,5 @@
 # -*- shell-script -*- :mode=shellscript:
 
 JFREECHART_HOME="$COMPONENT"
-JFREECHART_JAR_NAMES="iText-2.1.5.jar jcommon-1.0.18.jar jfreechart-1.0.14.jar"
-
-ISABELLE_JAVA_EXT="$ISABELLE_JAVA_EXT:$JFREECHART_HOME/lib"
+JFREECHART_JAR_NAMES="$(java-config -p itext | sed -e 's@:@ @g') $(java-config -p jcommon-1.0 | sed -e 's@:@ @g') $(java-config -p jfreechart-1.0 | sed -e 's@:@ @g')"
 
